sql-aggregates/profit-margin.sql

```diff
@@ -0,0 +1,9 @@
+select "title",
+      (sum("payments"."amount") - "replacementCost") as "totalProfit"
+  from "films"
+  join "inventory" using ("filmId")
+  join "rentals" using ("inventoryId")
+  join "payments" using ("rentalId")
```

Comment:
This `"replacementCost"` doesn't account for multiple copies of the same DVD in inventory. You'll want to calculate the total money spent on film inventory and then subtract that from all revenue collected per film.

Reply:
```plain
       title       |                                                    description                                                    | rating | totalProfit 
-------------------+-------------------------------------------------------------------------------------------------------------------+--------+-------------
 Titans Jerk       | A Unbelieveable Panorama of a Feminist And a Sumo Wrestler who must Challenge a Technical Writer in Ancient China | PG     |      105.79
 Maiden Home       | A Lacklusture Saga of a Moose And a Teacher who must Kill a Forensic Psychologist in A MySQL Convention           | PG     |      103.82
 Videotape Arsenic | A Lacklusture Display of a Girl And a Astronaut who must Succumb a Student in Australia                           | NC-17  |      101.78
 Kissing Dolls     | A Insightful Reflection of a Pioneer And a Teacher who must Build a Composer in The First Manned Space Station    | R      |       97.85
 Witches Panic     | A Awe-Inspiring Drama of a Secret Agent And a Hunter who must Fight a Moose in Nigeria                            | NC-17  |       96.77
(5 rows)


```